Client/Frontend/Reader/ReaderMode.swift

```diff
@@ -128,6 +117,36 @@ struct ReadabilityResult {
             return nil
         }
     }
+
+    /// Encode to a dictionary, which can then for example be json encoded
+    func encode() -> [String:AnyObject] {
```

Comment:
Nit: put this below `init?` so the initializers and `encode()` functions are grouped together.